server/views/pages/personSearchResults.njk

```diff
@@ -1,44 +1,56 @@
 {% extends "../partials/layout.njk" %}
 
-
```

Comment:
Is this also old code we don't need? Do we always paginate now?

Reply:
Nice catch! Removed!